Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/static/library/css/webkit.css

```diff
@@ -649,6 +649,18 @@ pre {
   border-radius: var(--tinySize);
 }
 
+.dot.tiny img, .dot.tiny .text {
+  width: calc(var(--tinySize) - 2px);
+  height: calc(var(--tinySize) - 2px);
+  font-size: calc(var(--tinySize) - 2px);
```

Comment:
This may make the font difficult to read, but I guess for this case it is only for symbols, it should be OK.